### ISSUE TITLE
Column Order list

### ISSUE THREAD

**Issue by Satej7** [swimlane/ngx-datatable#1552]
Can I get the sequence of columns in which they are ordered after drag and drop, because i need to save the order for the user.

**Comment by nage117** [comment]
Yes, I want to expand on this. Should emit the new columns (aka _internalColumns) along with the reorder and resize events so that you do not need to duplicate effort into calculating the very thing that the ngx-datatable has already calculated (or access what is likely meant to be a private property).

Can you rename issue to capture this larger scope: Emit new columns value (the value of _internalColumns) on reorder/resize.
